woof-code/rootfs-skeleton/usr/sbin/snapmergepuppy

```diff
@@ -204,12 +204,18 @@ do
 		if [ -L "$N" ] && [ -d "$BASE/$N" ] ; then
 			rm -rf "$BASE/$N"	# SFR: in case if folder has been replaced with a symlink (cp won't overwrite a dir with a symlink)
 		fi
-		cp -af "$N" "$BASE/$N" 2>>/tmp/snapmergepuppy-error
-		rm "$N"
+		#Move if the file was not opened by a process
+		if [ "$(lsof | awk '{ print $3 }' | grep "/" | grep -vE "\/dev|\/proc|\/sys|\/tmp|\/run|\/mnt|\/media" | sort | uniq | grep "$N")" == "" ]; then
```

Comment:
I think you mean `grep "^$N$"` - i.e. `/a/b/c` is not opened if the output contains `/a/b/cd`; only exactly `/a/b/c` means `/a/b/c` is open

```
~$ (echo abc; echo abcd) | grep abc
abc
abcd
~$ (echo abc; echo abcd) | grep '^abc$'
abc
```